dev/MRTCore/mrt/Microsoft.Windows.ApplicationModel.Resources/src/Microsoft.Windows.ApplicationModel.Resources.idl

```diff
@@ -113,4 +113,11 @@ namespace Microsoft.Windows.ApplicationModel.Resources
         static String TargetSize { get; };
         static String Theme { get; };
     }
-} // namespace Microsoft.Windows.ApplicationModel.Resources
+
+    [contract(MrtCoreContract, 1)]
+    runtimeclass ApplicationLanguages
+    {
+        static String PrimaryLanguageOverride;
```

Comment:
New API, so where's the test / sample in the peer Windows App SDK Samples repo?

Reply:
https://github.com/microsoft/WindowsAppSDK-Samples/pull/358